go.mod

```diff
@@ -158,6 +158,8 @@ require (
 	github.com/mailgun/minheap v0.0.0-20170619185613-3dbe6c6bf55f // indirect
 )
 
+require github.com/aclements/go-z3 v0.0.0-20220809013456-4675d5f90ca5
```

Comment:
if this is intentionally in its own require block separate from others, it probably deserves a comment here

Reply:
It's not, fixed